src/main/java/savvy/core/fact/events/DoSelectSubject.java

```diff
@@ -0,0 +1,9 @@
+package savvy.core.fact.events;
```

Comment:
UI events should be in their corresponding UI package
see: DoSelectFilterEntity
rename the event using the DoItemAction convention